docs/about.md

```diff
@@ -17,36 +17,24 @@ RepoSense is a project based in the [National University of Singapore, School of
 
 <!-- ------------------------------------------------------------------------------------------------------ -->
 
-### [Peh Xian Bin, Eugene](https://github.com/eugenepeh)
-![](https://avatars.githubusercontent.com/u/19277206?s=150&v=4)<br/>
-**Role**: Project Lead [2018 May - ]<br/>
-
-<!-- ------------------------------------------------------------------------------------------------------ -->
-
-### [Teng Yong Hao](https://github.com/yong24s)
-![](https://avatars2.githubusercontent.com/u/2003406?s=150&v=4)<br/>
-**Role**: Mentor [2018 Dec - ]<br/>
-Committer [2018 Aug - 2018 Dec]<br/>
-Contributor [2018 May - 2018 Aug]<br/>
-
-<!-- ------------------------------------------------------------------------------------------------------ -->
-
 ### [Wang Chao](https://github.com/fzdy1914)
 ![](https://avatars3.githubusercontent.com/u/35621726?s=150&v=4)<br/>
-**Role**: Committer [2019 May - ]<br/>
+**Role**: Project Lead [2021 Jan - ]<br/>
+Committer [2019 May - 2021 Jan]<br/>
 Contributor [2018 Dec - 2019 May]<br/>
 
 <!-- ------------------------------------------------------------------------------------------------------ -->
 
-### [James Pang Mun Wai](https://github.com/jamessspanggg)
-![](https://avatars1.githubusercontent.com/u/32864116?s=150&v=4)<br/>
-**Role**: Contributor [2019 May - ]<br/>
-
+### [Peh Xian Bin, Eugene](https://github.com/eugenepeh)
+![](https://avatars.githubusercontent.com/u/19277206?s=150&v=4)<br/>
+**Role**: Maintainer [2021 Jan - ]<br/>
+Project Lead [2018 May - 2021 Jan]<br/>
```

Comment:
Can put the end of my term to 2020 May. I have barely participated since then and you guys have put up great work taking over the responsibilities of RepoSense; hence the credits should go to both of you.